src/trace/ProxyTracerProvider.ts

```diff
@@ -35,10 +36,10 @@ export class ProxyTracerProvider implements TracerProvider {
   /**
    * Get a {@link ProxyTracer}
    */
-  getTracer(name: string, version?: string): Tracer {
+  getTracer(name: string, version?: string, options?: TracerOptions): Tracer {
     return (
-      this.getDelegateTracer(name, version) ??
-      new ProxyTracer(this, name, version)
+      this.getDelegateTracer(name, version, options) ??
+      new ProxyTracer(this, name, version, options?.schemaUrl)
```

Comment:
For consistency I would keep options 
```suggestion
      new ProxyTracer(this, name, version, options)
```